lib/rules/no-invalid-meta.js

```diff
@@ -46,6 +44,22 @@ module.exports = class NoInvalidMeta extends Rule {
             }
           }
         }
+        if (contentAttrValue.includes('user-scalable=no')) {
+          this.log({
+            message: 'a meta viewport should not restrict user-scalable',
+            line: node.loc && node.loc.start.line,
+            column: node.loc && node.loc.start.column,
+            source: this.sourceForNode(node),
+          });
+        }
+        if (contentAttrValue.includes('maximum-scale')) {
```

Comment:
This one is fine without custom parsing, since we don't care about the specific value